File: Basic/Operator/219158D.go
package main

import (
	"fmt"
)

func solution() {
	var a, b, c, d int

	fmt.Scanf("%d %d %d %d", &a, &b, &c, &d)
	fmt.Printf("Difference = %d\n", (a*b)-(c*d))
}

func main() {
	TEST_CASE := !true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}

File: Basic/Operator/219158D.c
#include <stdio.h>
#define intll long long

void solution() {
	int a, b, c, d;

	scanf("%d %d %d %d", &a, &b, &c, &d);
	printf("Difference = %lld\n", (1ll * a * b) - (1ll * c * d));

}

int main() {
	// char TEST_CASE = !1;
	int t = 1;
	// if (TEST_CASE == 1) {
	// 	scanf("%d", &t);
	// }

	while (t--) {
		solution();
	}
	return 0;
}